composer.json

```diff
@@ -16,9 +16,7 @@
         }
     ],
     "require": {
-        "php": ">=5.3.0"
-    },
-    "require-dev": {
+        "php": ">=5.3.0",
```

Comment:
Since we require ```symfony/process```, I'd suggest to require the same version (>=5.3.3) as symfony/process [does](https://github.com/symfony/Process/blob/2.1/composer.json#L19)